docs/adminguide/installation.rst

```diff
@@ -9,9 +9,11 @@ Install Linux Parsec CLI
 
 Parsec is also available as a command line interface (CLI) for Linux. It is a standalone binary that is provided in our GitHub releases.
 
+.. _Parsec CLI v3.0.2-a.0+dev: https://github.com/Scille/parsec-cloud/releases/download/v3.0.2-a.0+dev/parsec-cli_3.0.2-a.0+dev_linux_x86_64
```

Comment:
I'm not sure exactly who is the target of that doc, but if it's users and not just developers, I'd rather suggest installing the latest released version, not the currently developed one.

Reply:
You're right, but that already handle, when we will publish a new version that include this part of the documentation, the version will match the one released (that version is handled by `version_updater.py`).

Reply:
So for now this doc is not published ?

Reply:
Thanks !